test/gtest/ucp/test_ucp_am.cc

```diff
@@ -987,13 +993,13 @@ class test_ucp_am_nbx_send_copy_header : public test_ucp_am_nbx {
 
         ucs::fill_random(m_hdr_copy);
         while (progress());
-        wait_for_value(&cb_counter, req_counter);
+        wait_for_value(&m_recv_counter, m_send_counter);
```

Comment:
maybe add common wrapper function wait_receives() for "wait_for_value(&m_recv_counter, m_send_counter);"